### ISSUE TITLE
Rename driver/test namespaces in the PHP files

### ISSUE THREAD

**Issue by aik099** [minkphp/webdriver-classic-driver#17]
The namespaces of the PHP classes (both `src` and `tests` folders) need to be standardized to match other Mink drivers:

* namespace for the `src` folder: `Behat\Mink\Driver`;
* namespace for the `tests` folder: `Behat\Mink\Tests\Driver`.

**Comment by stof** [comment]
No. Using `Mink` as the top-level namespace was an explicit choice from me.

I won't go in the process of renaming the namespace from `Behat\Mink` to `Mink` in all existing package (the BC break for the ecosystem is not worth it), but I'd like to use `Mink` as top-level namespace for any new package going forward (as Mink is no longer under the Behat umbrella since a while, when we separated the organizations).
That's also why we use `mink/*` and not `behat/mink-*` for the composer package name.
